### ISSUE TITLE
Wallet resets when you reinstall the app

### ISSUE THREAD

**Issue by Pobepto** [Pobepto/walle#32]
If you reinstall the app, it deletes the old wallet

**Comment by khazhix** [comment]
Yes, I encountered the same problem, do you think data could be saved somewhere securely (cloud/pc storage)?
